### PR TITLE
Fix standardAppender when exception does not exist.

### PR DESCRIPTION
**Pull request by garrypolley**
This fixes issue #6 for Safari. This will also fix any other cases where exception is not passed to the standardLogAppender.

### CODE REVIEW

**Comment by sambao21** [comment]
could you test this change? and why not !exception as mentioned in #6?

**Comment by garrypolley** [comment]
I tested this change out. I didn't do the `!exception` before because I prefer explicit checks and `!exception` would give a truthy value if `exception` had a falsey value. 

If people generally prefer the `!exception` I can switch to that. I tried out both and testing resulted in the same.

**Comment by sambao21** [comment]
Being explicit is good. regarding testing, I was asking if there's a way to write a test for it. the current standard appender test is empty.

**Comment by garrypolley** [comment]
That was an oversight on my part. I noticed the no tests, and then went along. I'll try and get some tests added this weekend.

**Comment by garrypolley** [comment]
@sambao21 I added 2 tests for the default cases. I can add more tests later, but this gets us to at least testing the basics of the appender.

File: test/unit/appender/standard.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  describe('#standardLogAppender', function() {
+
+    it('returns correct keys when called', function () {
+      var actualAttributes = Canadarm.Appender.standardLogAppender(
+        Canadarm.level.INFO,
+        errorObject,
+        ''
+      );
+
+      assert(actualAttributes.characterSet);
```
Comment (sambao21) [inline test/unit/appender/standard.js]: is this just testing that attributes exist?

**Comment by sambao21** [comment]
i was imagining that you would be testing the 3 exception checks and asserting the scriptUrl. How does this test that?

File: test/unit/appender/standard.js (hunk 1 above)
Comment (garrypolley) [inline test/unit/appender/standard.js]: Yes this is verifying the attributes are present.

**Comment by garrypolley** [comment]
@sambao21 Added in 3 tests for the specific cases that were added.  Let me know if anything else is desired for this change.

**Comment by gneatgeek** [comment]
:+1:

**Comment by sambao21** [comment]
:+1:

**Pull request merged**